scripts/conda.sh

```diff
@@ -0,0 +1,9 @@
+echo "Creating conda env..."
```

Comment:
```suggestion
echo "Creating conda env `epispot`..."
```
Make sure to tell the user what environment is being created.